recipes/nose/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+rm -rf $PREFIX/man
+rm -rf $SP_DIR/man
+rm -rf $SP_DIR/nose-*.egg/man
+rm -f $PREFIX/bin/nosetests-*
```

Comment:
That is the same script as `nosetests` but with the python tag (`nosetest-2.7`). In conda's model it is useless, but if we one day can install two different Python versions in one `env`, then that is handy.